+nosnoc/+solver/MpccSolver.m

```diff
@@ -41,7 +41,7 @@
         ind_scalar_H
         ind_nonscalar_G
         ind_nonscalar_H
-        ind_map_G
+        ind_map_G % what is map_G?
```

Comment:
This is `(ind_map)_G` not `ind_(map_G)`. mildly confusing naming maybe but essentially it is used to track the mapping in case of lifting.

Reply:
this was my guess, please add a comment in the code. a reminder is helpful